pkg/network/event_windows.go

```diff
@@ -123,7 +122,53 @@ func FlowToConnStat(cs *ConnectionStats, flow *driver.PerFlowData, enableMonoton
 		if isFlowClosed(flow.Flags) {
 			m.TCPClosed = 1
 		}
+
+		if flow.ClassificationStatus == driver.ClassificationClassified {
+			switch crq := flow.ClassifyRequest; {
+			default:
+				// this is unexpected.  The various case statements should
+				// encompass all of the available values.
+
+			case crq == driver.ClassificationRequestUnclassified:
+				// do nothing because it may be classified in the response if
+				// the request portion of the flow was missed.
+
+			case crq >= driver.ClassificationRequestHTTPUnknown && crq < driver.ClassificationRequestHTTPLast:
+				cs.Protocol = ProtocolHTTP
+			case crq == driver.ClassificationRequestHTTP2:
+				cs.Protocol = ProtocolHTTP2
+			case crq == driver.ClassificationRequestTLS:
+				cs.Protocol = ProtocolTLS
+			}
+
+			switch crsp := flow.ClassifyResponse; {
+			default:
+				// this is unexpected.  The various case statements should
+				// encompass all of the available values.
+
+			case crsp == driver.ClassificationRequestUnclassified:
+				// do nothing because it will have been classified in the request
+
+			case crsp == driver.ClassificationResponseHTTP:
+				if flow.HttpUpgradeToH2Accepted == 1 {
+					// should we mark this as HTTP2 or HTTP?
```

Comment:
Do we know at this point? Or still an open question/want to leave this comment

Reply:
lol.  It was answered in a pvs. review to be as is.  Will remove comment.